core/src/main/kotlin/api/OrganizationsRoute.kt

```diff
@@ -232,8 +233,8 @@ fun Route.organizations() = route("organizations") {
                     val serviceName = call.requireParameter("serviceName")
                     val updateService = call.receive<UpdateInfrastructureService>()
 
-                    val updatedService = infrastructureServiceService.updateForOrganization(
-                        organizationId,
+                    val updatedService = infrastructureServiceService.updateForHierarchyId(
```

Comment:
Above you use `...ForHierarchy()`, here `...ForHierarchyId()`. I believe this should be aligned, ideally as described [above](https://github.com/eclipse-apoapsis/ort-server/pull/3022/files#r2165897075).